### PR TITLE
Fix #92: Search as you type delay

### LINKED ISSUE #92: Search as you type panel opens too late and first characters are not typed
When I click on the search bar and I immediately start typing, 90 % of the times the first characters do not get typed, as demonstrated here:

search-as-you-type-1

It happens to me a lot and it's quite annoying.

### PR DESCRIPTION
**Pull request by nienn**
Fixes #92

### CODE REVIEW

**Comment by astrojuanlu** [comment]
I tried "breaking" it several times and I couldn't, so +1 for me! Thanks @nienn 🚀

File: CHANGELOG.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+## Changes since 2021-08-17T07:33:29.505Z
+
+ * @stsewd: Make it easy to test locally (#83)
+
```
Comment (stsewd) [inline CHANGELOG.rst:4]: We usually run the automatic changelog command when doing a release

**Review by stsewd — COMMENTED** [review]
The other change looks good!

File: CHANGELOG.rst (hunk 1 above)
Comment (nienn) [reply CHANGELOG.rst:4]: Aha, yeah my bad. I was following the docs on "_[Local testing](https://another-test-stsewd.readthedocs.io/projects/readthedocs-sphinx-search/en/latest/development.html#local-testing)_" and continued following onto "_Releasing_" without realising 😬

Comment (nienn) [reply CHANGELOG.rst]: I rebased git history and removed this commit.

**Review by stsewd — APPROVED** [review]

**Comment by stsewd** [comment]
Hmm, tests failed, and then I run them again and now are passing... Maybe there is flaky test now, we should increase the wait time on our tests or find another way to avoid that wait. This was failing https://github.com/readthedocs/readthedocs-sphinx-search/blob/526eb581ccbca3d1364e843d27982d7716f699de/tests/test_ui.py#L146-L148

**Pull request merged**